src/evaluateXPath.ts

```diff
@@ -136,6 +149,101 @@ const evaluateXPath = <TNode extends Node, TReturnType extends keyof IReturnType
 		throw new TypeError("Failed to execute 'evaluateXPath': xpathExpression must be a string.");
 	}
 
+	const backend = options && options.backend ? options.backend : 'expression';
+
+	if (backend === 'js-codegen' || backend === 'auto') {
+		const compiledJavaScriptResult = compileWithJsCodegenBackend(selector, returnType, options);
+
+		if (compiledJavaScriptResult.isAstAccepted === true) {
```

Comment:
`if (a.b=== true)`  === `if (a.b)`

Reply:
Ah, TS garandeert natuurlijk dat het een boolean is.

Reply:
Done